# Statistical Analysis

This section of This section is created to analyze data statistically descriptively and inferentially. In order to determine the feasibility of Seblak Dropshipping Business

In [1]:
# Import Libraries 
import pandas as pd
import numpy as np
import time
from scipy import stats

In [2]:
# Import Clean Dataset
df = pd.read_csv('Seblak_tokped_clean.csv')

In [3]:
# Display Dataframe
df

Unnamed: 0                                               Name   Price  \
0            0  Gelifood Combo 3pcs Seblak Instan Kerupuk Mawa...  39.000   
1            1                                Kylafood Seblak Cup  11.000   
2            2  Seblak Instan Ceu Nthien Khas Bandung Rasana N...  15.000   
3            3                                SEBLAK INSTAN MOMMY  13.500   
4            4  Termurah Kerupuk Mentah-Kerupuk Seblak-Kerupuk...   6.000   
..         ...                                                ...     ...   
85          85  termurah cuanki lidah khas garut, toping baso ...  20.500   
86          86         kerupuk seblak pedas 1 kg / kerupuk seblak  35.625   
87          87    cuanki lidah 50 pcs enak toping seblak baso aci  12.999   
88          88                         PANGSIT SEBLAK BUMBU BASAH   8.500   
89          89  kerupuk seblak tatangga viral Rafael pedas dau...  10.000   

                      Seller             Cities  Product sold  Product rating  
0                   Lidigeli         Kab. Garut            30             4.6  
1           Kylafood Jakarta    Jakarta Selatan            60             4.9  
2   Central Seblak Nusantara  Tangerang Selatan          1000             4.9  
3           Rav Jaya Display     Kab. Tangerang          1000             4.8  
4           Djuragan Kerupuk           Surabaya            80             5.0  
..                       ...                ...           ...             ...  
85                 sansay327  Tangerang Selatan            30             5.0  
86         Adila_Putri_Snack         Kab.Ciamis           100             4.8  
87           CuankiCulametan         Kab. Garut           250             4.9  
88             RIKI STORE 97         Kab. Bogor             9             5.0  
89        MRA souvenir Ultah      Jakarta Timur             0             0.0  

[90 rows x 7 columns]

In [4]:
# Remove unnamed column
df.drop(columns='Unnamed: 0', inplace=True)

In [5]:
#The formula for calculating the Average of prices
print('Average Price:',df['Price'].mean(),)
#The formula for calculating the Median of price
print('Median of Price:',df['Price'].median(),)
#The formula for calculating the Mode of price
print('Mode of Price:',df['Price'].mode().values[0],)

Average Price: 21.474988888888888
Median of Price: 14.0
Mode of Price: 9.999



The average price of seblak on Tokopedia is Rp 21,474.99. This means that the majority of seblak products on Tokopedia are priced around this range.

The median price of seblak on Tokopedia is Rp 14.00. This means that half of the seblak products on Tokopedia are priced above Rp 14.00, and the other half are priced below Rp 14.00.

The mode price of seblak on Tokopedia is Rp 9.999. This means that the most common price for seblak on Tokopedia is Rp 9.999.

In [6]:
#The formula to calculate the skewness of a table
print(f"Skewness: {df['Price'].skew()}")

Skewness: 7.164824101503627


In [7]:
#The formula for calculating the kurtosis of a table
print(f"Kurtosis: {df['Price'].kurtosis()}")

Kurtosis: 59.69719180490647



The skewness of the price column of seblak is 7.164824101503627, which indicates that the distribution of the data is heavily skewed to the right. This means that there are a few very high-priced seblak products, and the majority of seblak products are priced lower.

The kurtosis of the price column of seblak is 59.69719180490647, which indicates that the distribution of the data is very peaked. This means that there is a concentration of seblak products priced at the mode price, and the distribution of prices falls off sharply on either side of the mode.

Overall, the skewness and kurtosis of the price column of seblak suggest that the distribution of seblak prices is very uneven, with a few very high-priced products and a concentration of products priced at the mode price.

Here are some possible interpretations of the results:

- Uneven distribution of prices: The skewness and kurtosis of the data suggest that the distribution of seblak prices is very uneven. This means that there is a small number of very high-priced seblak products, and the majority of seblak products are priced lower.
- Concentration of products at the mode price: The high kurtosis of the data suggests that there is a concentration of seblak products priced at the mode price. This means that consumers are more likely to find seblak products priced at the mode price than at other prices.
- Potential for price discrimination: The uneven distribution of seblak prices suggests that there is potential for price discrimination. Price discrimination is a practice where businesses charge different prices to different consumers for the same product or service. For example, a business might charge different prices to consumers based on their income level, location, or other factors.

Overall, the skewness and kurtosis of the price column of seblak suggest that the distribution of seblak prices is very uneven. This could have implications for consumers and businesses alike.

In [8]:
#The formula to calculate the Average of products sold
print('Average Product sold:',df['Product sold'].mean(),)
#The formula for calculating the Median of products sold
print('Median of Product sold:',df['Product sold'].median(),)
#The formula for calculating the Mode of product sold
print('Mode of Product sold:',df['Product sold'].mode().values[0],)

Average Product sold: 483.8333333333333
Median of Product sold: 80.0
Mode of Product sold: 100



The average product sold for seblak is 483.8333333333333. This means that the average seller on Tokopedia sells around 484 seblak products per month.

The median product sold for seblak is 80.0. This means that half of the sellers on Tokopedia sell more than 80 seblak products per month, and the other half sell fewer than 80 seblak products per month.

The mode product sold for seblak is 100. This means that the most common number of seblak products sold per month is 100.

These results suggest that there is a wide range of sales performance for seblak sellers on Tokopedia. Some sellers are selling a very large number of seblak products per month, while others are selling a much smaller number of products.

Here are some possible interpretations of the results:

- Wide range of sales performance: The wide range between the average, median, and mode product sold suggests that there is a wide range of sales performance for seblak sellers on Tokopedia. This is likely due to a number of factors, such as the seller's location, product selection, marketing strategy, and overall business experience.
- Potential for growth: The fact that the average product sold is higher than the median and mode product sold suggests that there is potential for growth for seblak sellers on Tokopedia. This is because there are a number of sellers who are not yet selling as many products as the average seller.
- Need for differentiation: The fact that the most common number of seblak products sold per month is 100 suggests that there is a need for differentiation among seblak sellers. Sellers who are able to differentiate themselves from the competition by offering unique products, services, or pricing strategies are more likely to be successful.

Overall, the results suggest that there is a wide range of sales performance for seblak sellers on Tokopedia. There is potential for growth for seblak sellers, but sellers need to differentiate themselves from the competition in order to be successful.

In [9]:
#The formula for calculating the Average of product ratings
print('Average Product rating:',df['Product rating'].mean(),)
#The formula for calculating the Median of product rating
print('Median of Product rating:',df['Product rating'].median(),)
#The formula for calculating the Mode of product rating
print('Mode of Product rating:',df['Product rating'].mode().values[0],)

Average Product rating: 4.45
Median of Product rating: 4.9
Mode of Product rating: 5.0



The average product rating for seblak on Tokopedia is 4.45. This means that the majority of seblak products on Tokopedia have a rating of around 4.5 stars.

The median product rating for seblak on Tokopedia is 4.9. This means that half of the seblak products on Tokopedia have a rating of 4.9 stars or higher, and the other half have a rating of 4.9 stars or lower.

The mode product rating for seblak on Tokopedia is 5.0. This means that the most common rating for seblak products on Tokopedia is 5 stars.

Overall, the data suggests that seblak products on Tokopedia are generally well-rated by consumers. The average, median, and mode product ratings are all relatively high.

In [10]:
#The formula to determine the upper boundary and lower boundary for column price
q1 = df['Price'].quantile(0.25)
q3 = df['Price'].quantile(0.75)
iqr = q3-q1

up_bound = q3 + 1.5*iqr
low_bound = q1 - 1.5*iqr

print(f'Upper Boundary: {up_bound:.2f}' )
print(f'Lower Boundary: {low_bound:.2f}')

Upper Boundary: 37.19
Lower Boundary: -6.31


This result show that the upper boundary for the price of seblak on Tokopedia is Rp 37.19 and the lower boundary is Rp -6.31. This means that any seblak product priced above Rp 37.19 or below Rp -6.31 would be considered an outlier.

In [11]:
#The formula to determine the upper boundary and lower boundary for the sold product column
q1 = df['Product sold'].quantile(0.25)
q3 = df['Product sold'].quantile(0.75)
iqr = q3-q1

up_bound = q3 + 1.5*iqr
low_bound = q1 - 1.5*iqr

print(f'Upper Boundary: {up_bound:.2f}' )
print(f'Lower Boundary: {low_bound:.2f}')

Upper Boundary: 587.12
Lower Boundary: -311.88


In the context of product sold, the upper and lower boundaries can be used to identify sellers that are selling an unusually high or low number of products. For example, if the upper boundary is 587.12 and the lower boundary is -311.88, then any seller that sells more than 587.12 products or less than -311.88 products would be considered an outlier.

In [12]:
#The formula for specifying the description of the database
df.describe()

Price  Product sold  Product rating
count   90.000000     90.000000       90.000000
mean    21.474989    483.833333        4.450000
std     34.557981   1551.701764        1.402105
min      2.881000      0.000000        0.000000
25%      9.999000     25.250000        4.800000
50%     14.000000     80.000000        4.900000
75%     20.875000    250.000000        5.000000
max    315.000000  10000.000000        5.000000

In this section, we want to determine the minimum and maximum potential of seblak sales revenue per month.

In [14]:
#The formula for creating a new column, the income column
df['Income']= df['Price'] * df['Product sold']
df

Name   Price  \
0   Gelifood Combo 3pcs Seblak Instan Kerupuk Mawa...  39.000   
1                                 Kylafood Seblak Cup  11.000   
2   Seblak Instan Ceu Nthien Khas Bandung Rasana N...  15.000   
3                                 SEBLAK INSTAN MOMMY  13.500   
4   Termurah Kerupuk Mentah-Kerupuk Seblak-Kerupuk...   6.000   
..                                                ...     ...   
85  termurah cuanki lidah khas garut, toping baso ...  20.500   
86         kerupuk seblak pedas 1 kg / kerupuk seblak  35.625   
87    cuanki lidah 50 pcs enak toping seblak baso aci  12.999   
88                         PANGSIT SEBLAK BUMBU BASAH   8.500   
89  kerupuk seblak tatangga viral Rafael pedas dau...  10.000   

                      Seller             Cities  Product sold  Product rating  \
0                   Lidigeli         Kab. Garut            30             4.6   
1           Kylafood Jakarta    Jakarta Selatan            60             4.9   
2   Central Seblak Nusantara  Tangerang Selatan          1000             4.9   
3           Rav Jaya Display     Kab. Tangerang          1000             4.8   
4           Djuragan Kerupuk           Surabaya            80             5.0   
..                       ...                ...           ...             ...   
85                 sansay327  Tangerang Selatan            30             5.0   
86         Adila_Putri_Snack         Kab.Ciamis           100             4.8   
87           CuankiCulametan         Kab. Garut           250             4.9   
88             RIKI STORE 97         Kab. Bogor             9             5.0   
89        MRA souvenir Ultah      Jakarta Timur             0             0.0   

      Income  
0    1170.00  
1     660.00  
2   15000.00  
3   13500.00  
4     480.00  
..       ...  
85    615.00  
86   3562.50  
87   3249.75  
88     76.50  
89      0.00  

[90 rows x 7 columns]

In [15]:
#Confidence interval formula to determine the minimum and maximum potential sales in column Income
std = df['Income'].std()
N = len(df)
low, up = stats.norm.interval(0.95,loc=df['Income'].mean(),scale=std/np.sqrt(N))
print('Lower Limit:',low)
print('Upper Limit:',up)

Lower Limit: 2099.0124726321915
Upper Limit: 7681.921816256696


From these results, it can be seen based on data calculations that the minimum potential income is around 2 million rupiah while the maximum income is around 7.6 million rupiah.

In this section, the program is made to determine whether the price of seblak in Jabodetabek is the same as the price of seblak outside Jabodetabek or more expensive since supplies outside the Jabodetabek area are usually cheaper.


So 2 hypotheses were made to find the answer to the question above

H0: The price of goods in Jabodetabek with prices outside Jabodetabek has no difference

H1: The price of goods in Jabodetabek with prices outside Jabodetabek has a big difference

In [16]:
# Formula to find the unique values of cities column
df['Cities'].unique()

array(['Kab. Garut', 'Jakarta Selatan', 'Tangerang Selatan',
       'Kab. Tangerang', 'Surabaya', 'Jakarta Timur', 'Jakarta Barat',
       'Bandung', 'Kab. Bekasi', 'Tangerang', 'Kab. Subang',
       'Jakarta Pusat', 'Medan', 'Depok', 'Malang', 'Kab. Tasikmalaya',
       'Cimahi', 'Bekasi', 'Kab. Bogor', 'Semarang', 'Tasikmalaya',
       'Kab.Ciamis'], dtype=object)

In [17]:
#The formula to determine the cities included in the Jabodetabek region
Kota = ['Tangerang Selatan', 'Jakarta Selatan', 'Kab. Bogor', 'Jakarta Timur', 'Jakarta Barat', 'Bekasi','Kab. Bekasi', 'Tangerang', 'Kab. Tangerang', 'Jakarta Pusat', 'Depok']
#The formula for creating a database that only contains cities in Jabodetabek
Jabodetabek = df[df['Cities'].isin(Kota)]

In [18]:
# formula to display Jabodetabek Dataframe
Jabodetabek.head()

Name  Price  \
1                                Kylafood Seblak Cup   11.0   
2  Seblak Instan Ceu Nthien Khas Bandung Rasana N...   15.0   
3                                SEBLAK INSTAN MOMMY   13.5   
5  Seblak RAFAEL | Seblak MAMANG RAFAEL | Seblak ...   40.0   
6                                      seblak frozen   10.4   

                     Seller             Cities  Product sold  Product rating  \
1          Kylafood Jakarta    Jakarta Selatan            60             4.9   
2  Central Seblak Nusantara  Tangerang Selatan          1000             4.9   
3          Rav Jaya Display     Kab. Tangerang          1000             4.8   
5    Naturally Plus Jakarta     Kab. Tangerang           100             5.0   
6       pelangi girls naget      Jakarta Timur          2000             4.9   

    Income  
1    660.0  
2  15000.0  
3  13500.0  
5   4000.0  
6  20800.0

In [19]:
#The formula to create a database that only contains cities outside Jabodetabek
Luar_Jabodetabek = df[~df['Cities'].isin(Kota)]

In [20]:
# formula to display outside of Jabodetabek area Dataframe
Luar_Jabodetabek.head()

Name   Price  \
0   Gelifood Combo 3pcs Seblak Instan Kerupuk Mawa...  39.000   
4   Termurah Kerupuk Mentah-Kerupuk Seblak-Kerupuk...   6.000   
14  KERUPUK SEBLAK RAFAEL KERING BANTAT PEDES DAUN...  16.000   
20           Seblak Instan Komplit Pedas Gurih Nikmat   5.999   
23  Kerupuk seblak pedas mawar daun jeruk (kerupuk...  17.000   

              Seller       Cities  Product sold  Product rating   Income  
0           Lidigeli   Kab. Garut            30             4.6   1170.0  
4   Djuragan Kerupuk     Surabaya            80             5.0    480.0  
14       AyundaSnack      Bandung            30             4.9    480.0  
20    Baso Aci Ayang   Kab. Garut          2000             4.9  11998.0  
23      snackmartbdg  Kab. Subang             0             0.0      0.0

In [21]:
#The formula for calculating the P value and T statistic between the price of goods in Jabodetabek and outside Jabodetabek
t_stat, p_val =stats.ttest_ind(Jabodetabek['Price'], Luar_Jabodetabek['Price'])

print('T-Statistics:', t_stat)
print('P Value:', p_val)

T-Statistics: 0.24244710841522868
P Value: 0.8089978968987732


From the results of this data, it can be concluded that the price of goods in Jabodetabek and outside Jabodetabek does not have a significant difference because of the results of the P value which is above 0.05, therefore H0 is accepted.

In this section, we want to find whether customers tend to prefer products with low prices or not?

To find out the answer to this question we determine the hypothesis, namely:

H0: Lower prices have no effect on total goods sold.

H1: Lower price has an effect on total goods sold.

In [22]:
#The formula for calculating the correlation between price and goods sold
corr_r, pval_p = stats.pearsonr(df['Price'], df['Product sold'])
print(f"r-correlation: {corr_r:.2f}, p-value: {pval_p}")

r-correlation: -0.10, p-value: 0.33064721935846364


Based on these results, it can be found that Price does not have a significant correlation with total goods sold because the P-Value result is above 0.05, therefore H0 is accepted.

# Conclusion

Based on the results of the data analysis, and the target of the SMART Framework. To achieve a sales result of 50 seblak per month, it is very possible for the median product sold to be at a value of 80 sold products and the mode to be at a value of 100. The target can be achieved or even exceeded as long as the promotional operations are also running well.

After that, in the context of finding suppliers, location does not have a major impact because the price of goods is not different between the Jabodetabek area and outside Jabodetabek, so it is still possible to find Jabodetabek suppliers at a reasonable price and also beneficial because it reduces shipping costs from a closer location.

In determining the price, it was also found based on the analysis that a cheap price does not guarantee more product sales. Therefore, we can determine the product price around the average price or even higher. Based on data calculations, it was also found that the minimum income potential is around Rp 3 million. Therefore, with a medium or high price in the future.

It can be concluded that the seblak business is a business that has great potential due to the cheap price of goods, making it possible for dropshipping businesses for additional income. With the goals set through SMART and the data available, the goals are very possible to be achieved within 3 months and the seblak sales operations can run smoothly and profitably.

### Additional explanation:

- SMART Framework: A goal-setting framework that stands for Specific, Measurable, Achievable, Relevant, and Time-bound.
- Jabodetabek: The Greater Jakarta metropolitan area, which includes Jakarta, Bogor, Depok, Tangerang, and Bekasi.
- Ongkir: Indonesian for "shipping costs".

### Closing Statement:

The author of the text concludes that the seblak dropshipping business is a viable business opportunity. The author's analysis shows that there is a high demand for seblak products, and that it is possible to achieve a gross sales of Rp 2 million per month with a medium or high price.